sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/model/SqlStmt.kt

```diff
@@ -45,25 +47,30 @@ fun ParserRuleContext.sqliteText(): String {
 }
 
 private fun ParserRuleContext.replacements(): Collection<Replacement> {
+
   if (this is SqliteParser.Type_nameContext && K_AS() != null) {
     return listOf(Replacement(K_AS().symbol.startIndex - 1, java_type_name().stop.stopIndex + 1, ""))
   }
   if (this is SqliteParser.Sql_stmtContext) {
     return listOf(Replacement(
-        sql_stmt_name().start.startIndex,
-        (getChild(2) as? ParserRuleContext)?.start?.startIndex ?: sql_stmt_name().stop.stopIndex,
-        ""
+            sql_stmt_name().start.startIndex,
+            (getChild(2) as? ParserRuleContext)?.start?.startIndex ?: sql_stmt_name().stop.stopIndex,
+            ""
     ))
   }
-  if (children == null) return emptyList()
-  return children.filterIsInstance<ParserRuleContext>().flatMap { it.replacements() }.toList()
+  var replacements:List<Replacement> = emptyList();
```

Comment:
Thanks.  I was going to ask about the way I've done the following few lines using `+=` - is that a reasonable approach?